LondonBritishLibrary/orient/BLorient4930.xml

```diff
@@ -0,0 +1,391 @@
+                                </msItem>
+                                <msItem xml:id="ms_i1.3.2">
+                                    <locus from="148ra"/>
+                                    <title ref="LIT2509Weddas#Tuesday"/>
+                                </msItem>
+                                <msItem xml:id="ms_i1.3.3">
+                                    <locus from="149vb"/>
+                                    <title ref="LIT2509Weddas#Wednesday"/>
+                                </msItem>
+                                <msItem xml:id="ms_i1.3.4">
+                                    <locus from="151vb"/>
+                                    <title ref="LIT2509Weddas#Thursday"/>
+                                </msItem>
+                                <msItem xml:id="ms_i1.3.5">
+                                    <locus from="154rb"/>
+                                    <title ref="LIT2509Weddas#Friday"/>
+                                </msItem>
+                                <msItem xml:id="ms_i1.3.6">
+                                    <locus from="155va"/>
+                                    <title ref="LIT2509Weddas#Saturday"/>
+                                </msItem>  
+                                <msItem xml:id="ms_i1.3.7">
+                                    <locus from="156rb"/>
+                                    <title ref="LIT2509Weddas#Sunday"/>
+                                </msItem>
+                            </msItem>
+                            <msItem xml:id="p1_i1.4">
+                                <locus from="158ra" to="162rb"/>
+                                <title ref="LIT1113Anqasa"/>
+                            </msItem>
+                        </msItem>
+                        <msItem xml:id="ms_i2">
+                            <locus from="124v" to="125r"/>
+                            <title ref="LIT4261Prayer"/>
+                            <incipit xml:lang="gez">እግዚአብሔር፡ እግዚእየ፡ ወአምላኪየ፡ እስእለከ፡ በፍቁር፡ ወልድከ፡ እግዚእነ፡ ኢየሱስ፡ ክርስቶስ፡ እግዚእየ፡ ወአምላኪየ፡ ወመድኃንየ፡
+                                ወበአስማቲከ፡ ቅዱሳት። ትስማዕ፡ ጸሎትየ፡ ወአስተበቍዖትየ። አርኅቅ፡ እምኔየ፡ ሰይጣነ፡ ወሠራዊቶ።</incipit>
+                        </msItem>
+                        <msItem xml:id="ms_i3">
+                            <locus target="#137v"/>
+                            <title ref="LIT6946InvocationMary"/>
+                            <incipit xml:lang="gez">ስብሐት፡ ለአብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ።   
+                                ለዓለም፡ ወለዓለመ፡ ዓለም።
+                                ሰአሊ፡ ለነ፡ ቅድስት፡ ድንግል፡ ማርያም። 
+                                ምሕረተ፡ ክርስቶስ፡ ወልድኪ፡ የሐውጸነ፡ እምአርያም።
+                                ኦአምላከ፡ ዳዊት፡ ወአምላከ፡ ነቢያት።
+                                ዕቀበኒ፡ ለገብርከ፡ <gap reason="illegible"/> እምዕለት፡ እኪት።</incipit>
+                        </msItem>
+                        <msItem xml:id="ms_i4">
+                            <locus target="#146v"/>
+                            <title>Two Salāms</title>
```

Comment:
If they take up c. 20 pages they are probably malkǝʾ-hymns rather than salams?

Reply:
The two pieces share the space on just one folio (146v) together with the end of LIT2362Songof#FifthCanticle. They are embedded to the Psalter, meaning that they are interspersed behind the Song of Songs and before Wǝddāse Māryām. If this is too confusing, if the numbers of msItem do not follow the order in the manuscript, it is probably more reasonable to see them as integral part of ms_i1 and list them as supplications or explicits inside ms_i1 (e.g. ms_i1.2 in that case), as I suggested above.

Reply:
Ah, I see. I wonder whether they might be later additions (we probably can't know without images). I would move this msItem between ms_i2 and ms_i3 (and change the numbering accordingly), if we assume they aren't additions

Reply:
also ms_i2 and ms_i3 should be moved to where they appear inside of the Psalter